src/AppInstallerTestExeInstaller/main.cpp

```diff
@@ -23,11 +23,21 @@ std::wstring GenerateUninstaller(std::ostream& out, const path& installDirectory
     path uninstallerOutputTextFilePath = installDirectory;
     uninstallerOutputTextFilePath /= "TestExeUninstalled.txt";
 
-    // TODO: Needs to re-invoke the installer and remove the Uninstall key that it added
+    path registryKey = "HKEY_CURRENT_USER" / path{ registrySubkey };

```

Comment:
>path registryKey [](start = 4, length = 16)

This is not a typical std::filesystem::path, I'd suggest constructing regkey as string